### ISSUE TITLE
Exception in CRUD/base_show.html.twig when admin.showTabs is false

### ISSUE THREAD

**Issue by ju1ius** [sonata-project/SonataAdminBundle#3726]
| Question       | Answer
|----------------|-------------------------------------------------------------
| Bundle version | dev-master
| Symfony version| 2.8.4
| php version    | 5.6.19

## Error message

> Impossible to access an attribute ("default") on a boolean variable ("") in [SonataAdminBundle:CRUD:base_show.html.twig at line 65](https://github.com/sonata-project/SonataAdminBundle/blob/master/Resources/views/CRUD/base_show.html.twig#L65)
> **500** Internal Server Error - **Twig_Error_Runtime**

## Steps to reproduce

Access the show route of an object when the admin class has no tabs configured (in my case the [CategoryAdmin](https://github.com/sonata-project/SonataClassificationBundle/blob/master/Admin/CategoryAdmin.php#L20) from SonataClassificationBundle).

## Expected results

I should either:
* see the `show` page with no fields
* see a more friendly error telling me to go and customize my configureShowFields method, or remove the show route.

## Actual results

The aforementioned exception.

## Easy guess

The [Admin::$showTabs](https://github.com/sonata-project/SonataAdminBundle/blob/master/Admin/Admin.php#L196) property defaults to false.
```php
/**
 * The view tab disposition.
 *
 * @var array|bool
 */
  private $showTabs = false;
```
When checking [the offending file](https://github.com/sonata-project/SonataAdminBundle/blob/master/Resources/views/CRUD/base_show.html.twig#L65), I see the following code, which was introduced by PR #3657 in [this commit](https://github.com/sonata-project/SonataAdminBundle/commit/fd1a1596f86bfb4afdafe9767efaa4c787bba6a5):
```twig
{% set has_tab = (admin.showtabs|length == 1 and admin.showtabs|keys[0] != 'default') or admin.showtabs|length > 1 %}
{% if has_tab %}
  ... stuff
{% else %}
  {{ show_helper.render_groups(admin, object, elements, admin.showtabs.default.groups, has_tab) }}
{% endif %}
```
We can see that the else block is expecting the admin.showTabs property to always have at least a default key, which doesn't seem to hold true.

**Comment by greg0ire** [comment]
@sidz, can you please  comment on this?

**Comment by muspelheim** [comment]
@greg0ire I also encountered this error today

**Comment by ju1ius** [comment]
@greg0ire I see that an [unrelated commit](https://github.com/sonata-project/SonataAdminBundle/commit/7f638a48b22d84b24e856903fdfd5e79e447c9ec?diff=unified#diff-0cb97fb894cd897069cbade4ba839c97R64) fixed the issue by doing this
```twig
{% elseif admin.showtabs is iterable %}
  {{ show_helper.render_groups(admin, object, elements, admin.showtabs.default.groups, has_tab) }}
{% endif %}
```
Shouldn't it be
```twig
{% elseif admin.showtabs.default.groups is defined %}
  {{ show_helper.render_groups(admin, object, elements, admin.showtabs.default.groups, has_tab) }}
{% endif %}
```

**Comment by sidz** [comment]
@greg0ire Sorry, I don't have a lot of time in this week, I saw that @ju1ius already fixed this issue via PR.

**Comment by ju1ius** [comment]
Closing since it was fixed in 7f638a48b22d84b24e856903fdfd5e79e447c9ec
See PR #3709
